### ISSUE TITLE
Peer Review of paper1

### ISSUE THREAD

**Issue by CSCmaster** [Yuki010305/paper-1#2]
There are a few problems you need to take considerations. First of all, you need to create three folds, input, output and script. The input should contains all the data you have cleaned and simulated. and the original data. The output folders should contains reference.bib, the final PDF file and the final qmd file. In the script folder, you should  put your 00simulated.R, 01_downloaddate.R, 02_cleandata.R. According to the schema, you should write the simulation of data virtually in an individual file and if you use Chatgpt please make sure to cite it. You should also  sketch your graphs and put it into an individual file.

```
ggplot(average_occupancy, aes(x = reorder(shelter_name, average_occupancy), y = average_occupancy, fill = shelter_name)) +
  geom_bar(stat = "identity") +
  coord_flip() + 
  labs(x = "Shelter Name", y = "Average Occupancy", title = "Top 10 Shelters by Average Occupancy") +
  theme_minimal() +
  theme(legend.position = "none") 
```

This code creates a horizontal bar plot using ggplot2, where the x-axis represents shelter names ordered by average occupancy, the y-axis represents average occupancy, and each bar is filled with a color corresponding to the shelter name. 
You need to explain your graph in details, for examples, where the maximum and minimum point occurs and trying to make connection of your research questions with your graph. 

```
filtered_data <- filtered_data %>% 
                 filter(!is.na(occupancy_date) & !is.na(occupancy))
```

This code removes rows from the filtered_data data frame where either the occupancy_date or occupancy column has missing values (NA). It ensures that the data frame only contains rows where both occupancy_date and occupancy have valid (non-missing) values.

```
ggplot(filtered_data, aes(x = occupancy_date, y = occupancy, color = shelter_name)) +
    geom_line() +
    theme_minimal() +
    labs(title = "Daily Shelter Occupancy: COSTI Reception Centre, Sojourn House, and Robertson House",
         x = "Date",
         y = "Occupancy")
```

This code creates a line plot using ggplot2. The x-axis represents dates, the y-axis represents daily shelter occupancy, and different shelters are distinguished by colored lines. The plot is styled with a minimalistic theme, and the title and axis labels are set accordingly. Also, you need to explain the graph in details, for examples, you need to explain what the x-axis and y-axis stand for and what are the characteristic point and are there any outliers in the graph.

```
ggplot(shelter_data, aes(x = shelter_city, y = occupancy)) +
  geom_boxplot() +
  coord_flip() +
  labs(title = "Spread of Occupancy in Different Shelters",
       x = "Shelter Name",
       y = "Occupancy") +
  theme_bw()
```

In conclusion, this code creates a horizontal boxplot using ggplot2. The x-axis represents shelter cities, the y-axis represents occupancy, and the boxes provide information about the spread of occupancy in different shelters. Also, you need to explain the boxplot in details.

Your paragraphs about data, graphs and conclusion are too short, you need to explain your data, graphs and conclusion about your theme in a very detailed ways and make sure to cite the R packages you have used and all the references you have used correctly. Also, generating the qmd file to PDf file is also very important, and create three folds and put all files into it so that your github page can be more layered.
